src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -283,7 +283,13 @@ def step(
             noise = torch.randn(
                 model_output.size(), dtype=model_output.dtype, layout=model_output.layout, generator=generator
             ).to(model_output.device)
-            variance = (self._get_variance(t, predicted_variance=predicted_variance) ** 0.5) * noise
+            if self.variance_type == "fixed_small_log":
```

Comment:
can we change the get_variance function to accommodate this? I think the fixed_small_log was added for this?